app/src/main/java/io/capibaras/abcall/data/repositories/IncidentsRepository.kt

```
@@ -0,0 +1,68 @@
+                        closedDate = getClosedDate(incident.history),
+                        recentlyUpdated = setRecentlyUpdated(incident)
+                    )
+                } ?: emptyList()
+
+                incidentDAO.refreshIncidents(incidents)
+
+                Result.success(incidents)
+            } else {
+                handleErrorResponse(localData, response.errorBody()?.string())
+            }
+        } catch (e: IOException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)